tox.ini

```diff
@@ -49,10 +49,15 @@ commands =
     bash {toxinidir}/locale/compile-mo.sh {toxinidir}/locale/
     pytest -n 2 -m 'needs_locales_compilation' -v src/olympia/ {posargs}
 
-[testenv:users-accounts-and-ratings]
+[testenv:users]
```

Comment:
Just realized that you also need to update `envlist` at the top of this file.

Reply:
https://github.com/mozilla/addons-server/pull/10450/commits/3cb082d8e15c25b1db7ec6be3658f70918626ed8